src/viur/core/bones/numeric.py

```diff
@@ -14,32 +14,34 @@
 MIN = -(sys.maxsize - 1)
 """Constant for the minimum possible value in the system"""
 MAX = sys.maxsize
-"""Constant for the maximum possible value in the system"""
+"""Constant for the maximum possible value in the system
+Also limited by the datastore (8 bytes). Halved for positive and negative values.
+Which are around 2 ** (8 * 8 - 1) negative and 2 ** (8 * 8 - 1) positive values.
+"""
 
 
 class NumericBone(BaseBone):
     """
-        A bone for storing numeric values, either integers or floats.
-        For floats, the precision can be specified in decimal-places.
-
-        :param precision: How may decimal places should be saved. Zero casts the value to int instead of
-            float.
-        :param min: Minimum accepted value (including).
-        :param max: Maximum accepted value (including).
+    A bone for storing numeric values, either integers or floats.
+    For floats, the precision can be specified in decimal-places.
     """
     type = "numeric"
 
     def __init__(
         self,
         *,
-        max: int | float = MAX,
         min: int | float = MIN,
-        mode=None,  # deprecated!
+        max: int | float = MAX,
```

Comment:
min before max and deprecated stuff at the end is more logical for my monk :)
Because they are keyword-only this doesn't matter